src/main/java/io/openliberty/tools/intellij/LibertyExplorer.java

```diff
@@ -220,6 +222,33 @@ public static Tree buildTree(Project project, Color backgroundColor) {
             }
         });
 
+        tree.addMouseMotionListener(new MouseMotionAdapter() {
+            private String currentTooltipText = null;
+
+            @Override
+            public void mouseMoved(MouseEvent e) {
+                super.mouseMoved(e);
+
+                TreePath path = tree.getPathForLocation(e.getX(), e.getY());
+                if (path == null ) {
+                    if (currentTooltipText != null) {
+                        tree.setToolTipText(null);
+                        currentTooltipText = null;
+                    }
+                } else {
+                    Object node = path.getLastPathComponent();
+                    if (node instanceof LibertyModuleNode) {
+                        LibertyModuleNode treeNode = (LibertyModuleNode) node;
```

Comment:
recommend renaming "treeNode" to "libertyNode" for clarity

Reply:
done.